Tilers/CityTiler/CityTilerDBConfig.yml

```diff
@@ -0,0 +1,5 @@
+PG_HOST: localhost
```

Comment:
Add this file to .gitignore so we avoid leaking our DB credentials to the rest of the world.